### ISSUE TITLE
Add mavlink camera trigger when GoPro not connected

### ISSUE THREAD

**Issue by Pedals2Paddles** [OpenSolo/shotmanager#9]
When there is no GoPro connected, the GoProManager.py `handleRecordCommand` function should send the mavlink `DO_DIGICAM_CONTROL` shutter command.  That way if the user has some other camera connected, the same controls can trigger that camera's shutter.  This will respond to the paddle button and the app shutter buttons.  The command should be inserted here: https://github.com/OpenSolo/shotmanager/blob/master/GoProManager.py#L386

**Comment by kellyschrock** [comment]
Good idea. I wonder though, does `DO_DIGICAM_CONTROL` work outside the context of a mission?

**Comment by Pedals2Paddles** [comment]
Yep.  It is also in GCS_Common.
https://github.com/ArduPilot/ardupilot/blob/671f39c8748b7dc4a62361d24ad41f10157e9a86/libraries/GCS_MAVLink/GCS_Common.cpp#L2055

**Comment by ivanskj** [comment]
Yes, this is exactly what I was looking for. Does anyone have a code example for sending DO_DIGICAM message?
